app/controllers/v1/pension_ipf_callbacks_controller.rb

```diff
@@ -63,7 +63,7 @@ def authenticity_error
     end
 
     def bearer_token_secret
-      Settings.dig(:pension_ipf_vanotify_status_callback, :bearer_token)
+      Settings.pension_ipf_vanotify_status_callback.bearer_token
```

Comment:
This exists [in the manifest repo](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/blob/main/apps/vets-api/staging/values.yaml#L2969-L2970) but not in vets-api settings.yml. 🤔 They were added to `config/settings/test.yml`, but I think it was an oversight that they weren't added to settings.yml. Can you do that as part of this PR? Or ask the original PR author to? https://github.com/department-of-veterans-affairs/vets-api/issues/15883